Open Judge System/Web/OJS.Web/Areas/Contests/Views/Results/Full.cshtml

```diff
@@ -21,99 +25,46 @@
-                <td>@participant.ParticipantFullName</td>
-                @foreach (var problemResult in participant.ProblemResults)
-                {
-                    <td style="white-space: nowrap">
-                        @if (problemResult.BestSubmission == null)
-                        {
-                            <span>@Resource.No_solution</span>
-                        }
-                        else
-                        {
-                            <strong><a href="/Contests/Submissions/View/@problemResult.BestSubmission.Id">@problemResult.BestSubmission.Points / @problemResult.MaximumPoints</a></strong> @problemResult.BestSubmission.SubmissionType
-                            <br />
-                            if (!problemResult.BestSubmission.IsCompiledSuccessfully)
-                            {
-                                @:Compilation failed!
-                            }
-                            foreach (var testRun in problemResult.BestSubmission.TestRuns)
-                            {
-                                var style = testRun.IsZeroTest ? "-ms-opacity: 0.3; opacity: 0.3" : string.Empty;
-                                switch (testRun.ResultType)
-                                {
-                                    case TestRunResultType.CorrectAnswer:<span style="@style" class="glyphicon glyphicon-ok text-success" title=@Resource.Answer_correct></span>break;
-                                    case TestRunResultType.WrongAnswer:<span style="@style" class="glyphicon glyphicon-remove text-danger" title=@Resource.Answer_incorrect></span>break;
-                                    case TestRunResultType.TimeLimit:<span style="@style" class="glyphicon glyphicon-time text-danger" title=@Resource.Time_limit></span>break;
-                                    case TestRunResultType.MemoryLimit:<span style="@style" class="glyphicon glyphicon-hdd text-danger" title=@Resource.Memory_limit></span>break;
-                                    case TestRunResultType.RunTimeError:<span style="@style" class="glyphicon glyphicon-asterisk text-danger" title=@Resource.Runtime_error></span>break;
-                                }
-                            }
-                        }
-
-                    </td>
-                }
-
-                <td>@participant.Total</td>
-            </tr>
-        }
-
-    </table>
-}
-            else
-            {
-                <h2>@Resource.NoMoreResults</h2>
-            }
-
-@if ((Model.CurrentPage == 1 && Model.Results.Count() == resultsOnPage) || Model.CurrentPage > 1)
-{
-    @Html.Partial("_SimplePagination", new SimplePaginationViewModel { HasNext = results.Count == resultsOnPage, CurrentPage = Model.CurrentPage, Url = "/Contests/" + isOfficial + "/Results/Full/" + Model.Id + "?page=" })
-}
\ No newline at end of file
+@Html.Partial("_FullPartial", Model)
```

Comment:
The name of the partial should be more expressive, currently its unclear what "_FullPartial" is exactly? Rename to something like "\_FullResultsPagedList" ( the "\_" at the start of the view name means its a partial view, so there is no need to manually write it in the name)

Reply:
Done in https://github.com/SoftUni-Internal/OpenJudgeSystem/pull/223